examples/python/linear_agent_example.py

```
@@ -0,0 +1,68 @@
+        "read_file",
+        "write_file",
+        "execute_command"
+    ],
+    memory=True,
+    web_search=True,
+    auto_approve_tools=True,
+)
+
+# Create Linear bot
+bot = LinearBot(
+    token="your-linear-oauth-token",  # or set LINEAR_OAUTH_TOKEN env var
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "token"

(S106)